src/conversion.js

```diff
@@ -48,7 +48,7 @@ export function createMetadata (remote: RemoteDoc): Metadata {
 
 // Extract the remote path and name from a local id
 export function extractDirAndName (id: string): [string, string] {
-  const dir = path.dirname(`/${id}`)
+  const dir = '/' + id.split(path.sep).slice(0, -1).join('/')
```

Comment:
It does work, but as stated in the method comment, the goal is to extract the remote path, which uses slashes as separators.